### DISCUSSION TITLE
댓글 신고 기능 정책에 대해서

### DISCUSSION THREAD

**Discussion by devitssu** [one-day-one-meal/one-hour-one-meal#73] [Ideas]
1. 현재 댓글 신고 시 중복 신고가 안되도록 막고 있습니다.
그리고 관리자가 댓글 신고를 무시하면 report rowr가 사라집니다. 따라서 이후 같은 사용자가 같은 댓글에 신고하면 정상적으로 신고가 됩니다.
이럴 경우를 허용할까요 아니면 report를 soft delete 해서 안 되도록 막는게 좋을까요?

2. 관리자가 신고된 댓글을 처리할 때 해당 댓글과 관련된 신고를 모두 지우는게 좋을지 아니면 soft delete를 하고 차단된 댓글일시 `신고로 차단된 댓글입니다.`와 같은 메시지를 보여주는게 좋을지 고민이 됩니다.

**Comment by mobzzzzz** [comment]
Soft delete 적용하면 정책적으로 가장 어울리는 테이블이 될 것 같고
무시는 그냥 무시인게 사실 고객센터 평균 아닐까 싶습니다
